lxd/auth/oidc/oidc.go

```diff
@@ -296,10 +296,14 @@ func getAccessTokenVerifier(issuer string) (op.AccessTokenVerifier, error) {
 }
 
 // NewVerifier returns a Verifier.
-func NewVerifier(issuer string, clientid string, audience string) *Verifier {
-	cookieKey := []byte(uuid.New())[0:16]
+func NewVerifier(issuer string, clientid string, audience string) (*Verifier, error) {
+	cookieKey, err := uuid.New().MarshalBinary()
```

Comment:
Before this was using the first 16 bytes from the hex encoded UUID string (128 bytes in total). Now it's using all of the UUIDs non hex encoded 16 bytes for the `cookieKey`.

This shouldn't be a breaking change but to confirm the OIDC integration is still working I have deployed a Keycloak and added a new remote using `--auth-type oidc`.

Reply:
Oh good spot. This should previously have been using `uuid.NewRandom().Array()`. The google package is much improved in that regard.